pages/index.tsx

```diff
@@ -115,29 +112,46 @@ const ConnectWithMe = () => {
     <div className="flex items-center h-fit bg-gray-200/40 backdrop-blur-sm border-2 border-zinc-100 w-fit  px-2 py-2  rounded-tl-full rounded-bl-full pr-10 translate-x-40 hover:translate-x-6  transition">
       {/* linkedin connect button with icon  */}
       <div className="flex flex-row m-auto items-center space-x-4">
-      <Image className="rounded-full" alt="LinkedIn" width="60" height="60" src="https://avatars.githubusercontent.com/u/79275157?v=4"/>
-        <div onClick={() => 
-    // go to linked in page
-    window.location.href="https://www.linkedin.com/in/piyush-kalyan/"
-    } className="cursor-pointer" >
-        <span className="flex flex-row w-fit gap-2 bg-[#2885e0] p-2 text-white font-medium rounded-full">
-          <div>
-          <LinkedInIcon/>
-          </div>
-          <a  href="https://www.linkedin.com/in/piyush-kalyan/">Linked In</a>
-        </span>
-   
+        <Image
+          className="rounded-full"
+          alt="LinkedIn"
+          width="60"
+          height="60"
+          src="https://avatars.githubusercontent.com/u/79275157?v=4"
+        />
+        <div
+          onClick={() =>
+            // go to linked in page
+            (window.location.href =
+              "https://www.linkedin.com/in/piyush-kalyan/")
+          }
+          className="cursor-pointer"
+        >
+          <span className="flex flex-row w-fit gap-2 bg-[#2885e0] p-2 text-white font-medium rounded-full">
+            <div>
+              <LinkedInIcon />
+            </div>
+            <a href="https://www.linkedin.com/in/piyush-kalyan/">Linked In</a>
+          </span>
```

Comment:
There are no changes in code, please revert back the changes.

Reply:
I have done no code changes here, there was just auto indentation (prettier extension) in vs code which indented the code in neat lines according to it's algo.